call/bitrate_allocator.cc

```diff
@@ -379,6 +379,10 @@ void BitrateAllocator::OnNetworkEstimateChanged(TargetTransferRate msg) {
   RTC_DCHECK_RUN_ON(&sequenced_checker_);
   last_target_bps_ = msg.target_rate.bps();
   last_stable_target_bps_ = msg.stable_target_rate.bps();
+
+  // Try to underutilize the estimated bandwidth to reduce overshoots.
+  last_target_bps_ = last_target_bps_ * 0.9;
```

Comment:
Can we put the multiplier in api/transport/bitrate_settings.h as well?  That way the numbers are near each other.